protoc-gen-scheme

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env swish
```

Comment:
What's the plan to make this work in web? The web docker image doesn't have swish installed.

Reply:
See https://github.com/indigobio/ascent-web/blob/protobuf/referee/src/protoc-gen-scheme.in